### PR TITLE
Add metrics for compression ratio at different levels

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by javaforfun** [comment]
LGTM

File: src/util/rocksdb/engine_metrics.rs [hunk 1]
```diff
@@ -850,6 +852,17 @@ pub fn flush_engine_properties(engine: &DB, name: &str) {
                 .with_label_values(&[name, cf])
                 .set(pending_compaction_bytes as f64);
         }
+
+        // Compression ratio at levels
+        let opts = engine.get_options_cf(handle);
+        for level in 0..opts.get_num_levels() {
```
Comment (zhangjinpeng1987) [inline src/util/rocksdb/engine_metrics.rs]: Some levels maybe empty, is it ok?

Comment (huachaohuang) [reply src/util/rocksdb/engine_metrics.rs]: Yep, the compression ratio of an empty level will be -1.0.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**